test/react/render-props/react-root-context.js

```diff
@@ -0,0 +1,47 @@
+var React = require('React');
+// the JSX transform converts to React, so we need to add it back in
+this['React'] = React;
+
+var { Provider, Consumer } = React.createContext(null);
```

Comment:
Let's put something more interesting as default value? To ensure we don't accidentally use null for some other reason.